src/tests/Common/Platform/platformdefines.h

```diff
@@ -88,8 +88,6 @@ typedef unsigned int ULONG, *PULONG;
 #define __int64     long long
 #endif // HOST_64BIT
 
-#define UInt32x32To64(a, b) ((unsigned __int64)((ULONG)(a)) * (unsigned __int64)((ULONG)(b)))
```

Comment:
I think this should be reverted.

Reply:
This isn't currently in use, and we may not use it in this form at all.